gems/quilt_rails/README.md

```diff
@@ -300,6 +300,67 @@ function App() {
 export default App;
 ```
 
+##### Example: sending headers from Rails controller
+
+```ruby
+class ReactController < ApplicationController
+  include Quilt::ReactRenderable
+
+  def index
+    render_react(headers: { 'x-custom-header': 'header-value-a' })
+  end
+end
+```
+
+You will need to serialize the result of the useRequestHeader hook in order for it to persist to the client
+
+```tsx
+// app/ui/foundation/CustomUniversalProvider.tsx
+import {createContext} from 'react';
```

Comment:
@cartogram you seemed to be under the impression we were handling this automagically now

Reply:
@cartogram just wanted to check again to see if you had any concerns about this part? I couldn't get the `useRequestHeader` to persist without serializing it separately but if there is a way to do it correctly I can remove this section.

Reply:
I still haven't run into issues with the `useRequestHeader` when testing in a few projects we use it, but it does seem like this might be necessary for this use.